src/components/NewClusterModal.js

```diff
@@ -773,6 +773,7 @@ export const NewClusterModal = withModalDrawer({ width: 675 })(class NewClusterM
                   id,
                   isSearchable: false,
                   value: sparkMode,
+                  menuPosition: 'fixed',
```

Comment:
This introduces a different problem. If the pane is scrollable and you open the select, the options don't scroll with the pane. Is there another way we can fix this?

Reply:
The only other way I could think of to do it was to put was `menuPlacement: 'top'`, but that's a change in behavior.

Reply:
We could add `flex: 'auto'` to the scrollable div to have it expand to fill the space, which should fix it.

Reply:
I will try that!

Reply:
That solves it, pushing up shortly.